### PR TITLE
Add back button to preview page

### PR DESCRIPTION
**Pull request by gjvis**
Still to do:
- CSS
- Javascript needs to be properly initialised from the form on the new appointment summary page

### CODE REVIEW

File: features/support/pages/preview_page.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
-class RecordOfGuidancePreviewPage < SitePrism::Page
+class PreviewPage < SitePrism::Page
```
Comment (stevenwilkin) [inline features/support/pages/preview_page.rb]: Good call on renaming this

File: app/views/appointment_summaries/_preview_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<%= form_for local_assigns.fetch(:appointment_summary),
+  url: local_assigns.fetch(:url),
+  method: local_assigns.fetch(:method) { :post } do |f| %>
+
+  <% (AppointmentSummary.columns.map(&:name) - [:id, :created_at, :updated_at, :user_id, :name, :address]).each do |field| %>
```
Comment (stevenwilkin) [inline app/views/appointment_summaries/_preview_form.html.erb]: I've raised #90 in response to presence of `name` and `address` here

Comment (gjvis) [inline app/views/appointment_summaries/_preview_form.html.erb]: Cheers :+1:

File: app/views/appointment_summaries/_preview_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <%= f.hidden_field field %>
+  <% end %>
+
+  <%= f.submit local_assigns.fetch(:submit_button_text),
+    class: (%w(btn btn-lg) | Array(local_assigns[:submit_button_classes])) %>
```
Comment (stevenwilkin) [inline app/views/appointment_summaries/_preview_form.html.erb]: Nice use of set union

**Comment by stevenwilkin** [comment]
looks good

File: app/views/appointment_summaries/_preview_form.html.erb (hunk 2 above)
Comment (stevenwilkin) [inline app/views/appointment_summaries/_preview_form.html.erb]: `name` and `address` have since been removed, FYI

Comment (gjvis) [inline app/views/appointment_summaries/_preview_form.html.erb]: Doh >_<

**Comment by aduggin** [comment]
@gjvis please rebase

**Comment by gjvis** [comment]
@stevenwilkin can you handle the rebase on this please?

**Comment by stevenwilkin** [comment]
Rebased!

**Pull request merged**